.github/workflows/firefox-ios-autofill-playwrite-tests.yml

```diff
@@ -0,0 +1,41 @@
+name: Build and Run Autofill Automation
+permissions: read-all
+on:
+    pull_request:
+      paths:
+        - 'firefox-ios/Client/Assets/CC_Script/**'
+    push:
+      branches: ['main']
+      paths:
+        - 'firefox-ios/Client/Assets/CC_Script/**'
```

Comment:
Note: 
a) Do the playwrite folder also live here?
b) Will this PR run anytime we make a change on that path?

@isabelrios I know the answer to b) but was wondering if you have any thought around this